cmd/main.go

```diff
@@ -83,7 +85,8 @@ func main() {
 		log.Fatal().Err(err).Msg("Unable to create tls cert handler")
 	}
 
-	svcHandler := est.NewStaticServiceHandler(est.NewService(rootCert, caCert, caKey))
+	duration := time.Hour * 24 * time.Duration(*durationDays)
```

Comment:
Do I read it correctly that shortest possible duration is 1 day?

Reply:
hmm... interesting question.

@doanac I think we can allow `cert-duration-days` to accept either a float or a generic _time duration_ format.
https://pkg.go.dev/time#ParseDuration

Reply:
i've changed it to accept golang's time.Duration format:

 https://cs.opensource.google/go/go/+/refs/tags/go1.21.5:src/time/format.go;l=1585-1588